index.html

```diff
@@ -0,0 +1,64 @@
+<html>
+<head>
+  <title>The Lorem Micro Blog</title>
+  <link rel="stylesheet" type="text/css" href="styles.css"/>
+</head>
+<body>
+  <nav>
+    <h1>
+      The Lorem Micro Blog
```

Comment:
What's the plan for using this "The Lorem Micro Blog" both in nav and header? Are you using Nav for the tan bit going across the screen?